src/docs/product/performance/getting-started.mdx

```diff
@@ -17,12 +17,17 @@ Customers on legacy plans must add transaction events to their subscription in o
 
 ## Supported SDKs
 
+- [Java](/platforms/java/performance/)
+
 - [JavaScript](/platforms/javascript/performance/)
   - [Angular](/platforms/javascript/guides/angular/performance/)
   - [Ember](/platforms/javascript/guides/ember/performance/)
   - [React](/platforms/javascript/guides/react/#monitor-performance)
   - [Vue](/platforms/javascript/guides/vue/#monitor-performance)
 
+- Mobile
```

Comment:
@bruno-garcia not sure about a Mobile section here but under Java would be kinda weird if we now have a Kotlin language, wdyt?

Reply:
Make sense to me to categorize like you did. We could rename Java to [Java/Kotlin/JVM] ?